keps/sig-node/2570-memory-qos/README.md

```diff
@@ -104,6 +107,62 @@ Brief map as follows:
 | pod | sum(requests.memory) | n |
 | node | n/a | pods, kube-reserved, system-reserved | n |
```

Comment:
There is something wrong with this table, it should be:

| type | memory.min | memory.high |
| -------- | -------- | -------- | 
| container | requests.memory | limits.memory/node allocatable memory * memory throttling factor |
| pod | sum(requests.memory) | n/a |
| node | pods, kube-reserved, system-reserved | n/a |

@ndixita Could you please update it by the way?

Reply:
When is memory.high not set for pod or node? I am unable to follow the case here